packages/core/src/virtual-entrypoint.ts

```diff
@@ -312,9 +312,18 @@ function shouldSplitRoute(routeName: string, splitAtRoutes: (RegExp | string)[]
   }
   return splitAtRoutes.find(pattern => {
     if (typeof pattern === 'string') {
+      if (pattern.startsWith('/') && pattern.endsWith('/')) {
+        let fragments = pattern.match(/\/(.*?)\/([a-z]*)?$/i);
```

Comment:
this was taken from a few places on stack-overflow and other blog posts on the topic. I'm sure this is probably overkill but it seems to work fine 🤷